source/common/res/features/running-balance/main.js

```diff
@@ -108,12 +101,9 @@
       function onYnabGridyBodyChanged() {
         var accountController = ynabToolKit.shared.containerLookup('controller:accounts');
 
-        if (accountController.get('sortProperties').indexOf('date') !== -1) {
-          updateRunningBalanceCalculation(accountController.get('arrangedContent'));
-          updateRunningBalanceColumn();
-        } else {
-          $('.ynab-toolkit-grid-cell-running-balance').remove();
-        }
+        sortContent(accountController.get('content'));
```

Comment:
rut roh! 

We need the full array to properly calculate the running balance which is why `content` was. I guess I could use the `transactionDisplayItems` to get this same data -- basically as long as I can get every transaction on the account, I should be fine here.